assets/facets.js

```diff
@@ -7,12 +7,40 @@ class FacetFiltersForm extends HTMLElement {
       this.onSubmitHandler(event);
     }, 500);
 
-    this.querySelector('form').addEventListener('input', this.debouncedOnSubmit.bind(this));
+    const facetForm = this.querySelector('form');
+    facetForm.addEventListener('input', this.debouncedOnSubmit.bind(this));
+    facetForm.addEventListener('keydown', (event) => {
+      if (event.target.tagName === 'LI' && (event.code.topUpperCase() === 'SPACE' || event.code.toUpperCase()  === 'ENTER' )) {
+        event.preventDefault();
+        event.target.querySelector('input').click();
+      }
+    });
+
+    facetForm.addEventListener('click',  (event) => {
+      if (event.target.nodeName === 'SPAN' && event.target.closest('[id^="Show-More-"]')) {
+        this.expandShowMore(event.target.closest('[id^="Show-More-"]'));
+      }
+    });
+
+    facetForm.addEventListener('keydown',  (event) => {
+      if (event.target.className === 'button-show-more link underlined-link' && (event.code.toUpperCase() === 'SPACE' || event.code.toUpperCase() === 'ENTER' ) && event.target.closest('[id^="Show-More-"]')) {
```

Comment:
Since this is the only button I updated it to be the node type. What do you think? https://github.com/Shopify/dawn/pull/1443/commits/2f3b9363db2f48dffa2093bdb857acd34c37c1e2

Reply:
That is fair 🤔 However it's a good point that class names can change. I think checking for the node type (button) makes more sense. Thoughts?

Reply:
Yeah in this case it's fine since the component is small and only has 1 button. But yeah whether we should ever reference classes in JS would be a good point to bring up in team discussion.